.github/actions/generate-test-dbs/action.yml

```diff
@@ -0,0 +1,27 @@
+name: "generate-test-dbs"
+description: "Generate qcodes dbs used for upgrade tests"
+runs:
+  using: "composite"
+  steps:
+    - name: generate test dbs
+      run: |
+        pip install -e .
+        pwd
+        cd ..
+        git clone https://github.com/QCoDeS/qcodes_generate_test_db.git
```

Comment:
is it more beneficial to use action's checkout (or clone?) step instead of `git clone ...` manually?

Reply:
It's not currently possible to use actions from whinin a composable run step as I understand. So I suggest to leave as is for now